### PR TITLE
converted visual to new API

### PR DESCRIPTION
**Pull request by mvgaliev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## 1.3.0
```
Comment (ignatvilesov) [inline CHANGELOG.md]: Version must be the same as specified in legacy visual.

File: capabilities.json [hunk 2]
```diff
@@ -178,6 +220,310 @@
...
+                "isItalic": {
+                    "displayName": "Italic",
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "fontFamily": {
+                    "displayName": "Font family",
+                    "type": {
+                        "enumeration": [
```
Comment (ignatvilesov) [inline capabilities.json]: Are you sure that all of fonts work well in the sandbox mode?

File: src/visual.ts [hunk 3]
```diff
@@ -436,17 +407,39 @@ namespace powerbi.extensibility.visual {
...
+
+            if (width < 450 || height < 450) {
+                return DualKpiSize.medium;
+            }
+
+            return DualKpiSize.large;
         }
 
         public update(options: VisualUpdateOptions) {
+            debugger;
```
Comment (ignatvilesov) [inline src/visual.ts]: Please remove

File: style/visual.less [hunk 4]
```diff
@@ -70,11 +69,13 @@
     }
 
     .data-title {
-        font-family: @regularSecondaryFontFamily;
-        font-size: 10px;
         text-transform: uppercase;
     }
 
+    .data-title.extra-small {
```
Comment (ignatvilesov) [inline style/visual.less]: Don't we use formatting utils any more?

File: CHANGELOG.md (hunk 1 above)
Comment (uve) [reply CHANGELOG.md]: @ignatvilesov why?

Comment (ignatvilesov) [reply CHANGELOG.md]: Legacy version is `1.3.4`. Change log shows `1.3.0`. That's a bit confused.

The version must be the same as a legacy or greater.

File: capabilities.json (hunk 2 above)
Comment (mvgaliev) [reply capabilities.json]: all specified fonts works fine

File: style/visual.less (hunk 4 above)
Comment (mvgaliev) [reply style/visual.less]: Don't think this comment has high priority before new API visual will be approved

Comment (ignatvilesov) [reply style/visual.less]: You removed the import of formatting utils' `CSS` styles.
I'm wondering why you did so?

That's ok if Dual KPI doesn't use Formatting Utils but Formatting Utils is included into TS and JS code.

**Pull request merged**